python/ctsm/subset_data.py

```diff
@@ -571,7 +571,11 @@ def setup_files(args, defaults, cesmroot):
         abort("inputdata directory does not exist")
 
     # DATM data
-    datm_type = "datm_gswp3"
+    # TODO issue #1895: allow datm_crujra, which also affects
```

Comment:
Change TODO to say "make this option available at the command line"

Reply:
Other TODOs
- [ ] Update .gitmodules when cdeps tag is ready
- [ ] Update ChangeLog/Sum